src/components/AuthContainer.vue

```diff
@@ -0,0 +1,74 @@
+<template lang="pug">
+  form.auth-container(@submit.prevent)
+    img.logo(src='../assets/logo-wolox.png')
+    slot(name='content')
+    button.button.primary.sign-up-button(type='submit' :disabled="!valid" @click.prevent="primaryAction") {{ primaryTitle }}
+    router-link.button.secondary(type='button' :to='alternativeRoute') {{ alternativeRouteTitle }}
+    span.request-error(:v-show='!!error') {{ error }}
+</template>
+
+<script>
+export default {
+  name: 'AuthContainer',
+  props: {
+    primaryAction: { type: Function, required: true },
+    primaryTitle: { type: String, required: true },
+    alternativeRoute: { type: String, required: true },
+    alternativeRouteTitle: { type: String, required: true },
+    valid: { type: Boolean, default: true },
+    error: { type: String }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+@import '../scss/application';
```

Comment:
This should only be imported in the `App` component without the `scoped` so in the component you only import the modules you need. For example, only import colors or sizes

Reply:
Fixed it in another PR